irispy/iris_tools.py

```diff
@@ -211,10 +221,71 @@ def get_iris_response(pre_launch=False, response_file=None, response_version=Non
         del(iris_response["DATE"])
     return iris_response
 
+def fit_iris_xput(time_obs, time_cal_coeffs, cal_coeffs):
 
```

Comment:
A docstring is required here explaining what the inputs and outputs are.  This will take some investigation.

Reply:
Added a draft docstring in. Please review to check for correctness.

Reply:
Added a draft docstring that needs reviewing to ascertain correctness.